.github/workflows/integration-test.yml

```diff
@@ -14,6 +14,8 @@ jobs:
         - macos-latest
         - windows-latest
     runs-on: ${{ matrix.platform }}
+    needs:
+    - prepare
```

Comment:
This workflow does not have a job called "prepare". Would that be necessary though?